internal/faustjs.org/docs/migrationPath/overview.mdx

```diff
@@ -2,6 +2,279 @@
 slug: /migrationPath/overview
 title: Migration from Legacy Faust
 ---
-Migrate from old version of Faust.js to new Faust
 
-### This document is in progress. Check back soon for content!
\ No newline at end of file
+## General Guidelines
+
+Currently, there is no great automated way to migrate from previous versions of Faust that use GQty to the recent version using Apollo. This is primarily due to their striking differences in the way the queries are performed.
+However, there are some conventional techniques and best practices for React Development that will definitely help you with this process.
+
+### Reusing Presentational Components
+If you paid attention while developing your Faust website, the Components folder typically contains components that embody a presentational meaning.
```

Comment:
I wouldn't say things like "If you paid attention", that seems a little rude or condescending.